<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Nick Crews</author>
 <class>GUI</class>
 <widget class="QMainWindow" name="GUI">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>637</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Hot Topic</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="Line" name="dividerLine">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="inputLabel">
        <property name="text">
         <string>Dataset:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QListView" name="datasetList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="datasetView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="modelLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Model:</string>
          </property>
          <property name="buddy">
           <cstring>modelBrowseButton</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="modelLineEdit">
          <property name="text">
           <string>None set...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="modelBrowseButton">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="predictButton">
        <property name="text">
         <string>Predict!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="display">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string>Display</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </action>
  <action name="actionKg">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>kg</string>
   </property>
  </action>
  <action name="actionLbs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>lbs</string>
   </property>
  </action>
  <action name="actionOpenCal">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionSaveCal">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveCalAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpenRec">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSaveRec">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveRecAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionExportSnippet">
   <property name="text">
    <string>Export Snippet...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
